man/elogv.1
.TH ELOGV "1" "May 2008" "User Commands"
.SH NAME
elogv \- elog viewer
.SH DESCRIPTION
Elogv is a portage elog viewer based on curses and python.
.SH USAGE
You can use these keys to control the behavior of the program:
.TP
.B "Down arrow" or "j"
Scroll the list of files down by 1 unit
.TP
.B "Up arrow" or "k"
opposite of Down arrow
.TP
.B "PagDown"
Scroll the list down by 10 unit
.TP
.B "PagUp"
opposite of Pag Down
.TP
.B End
Go to the last file of the list
.TP
.B Home
Go to the first file of the list
.TP
.B t
order the list of files by date, most recent on top
.TP
.B a
order the list of files alphabetically, the first time by category, the second
time (pressing the key again) by package name
.TP
.B c
order the list of files by log class warning level
.TP
.B r
reverse the list of files
.TP
.B SpaceBar
scroll the selected file
.TP
.B  h or F1
show the help screen, press Page Up/Down to scroll up and down, h and F1 again to hide
.TP
.B d
removes selected files, usage is similar to vim "d" command, here are same examples:
.RS
.TP
.I da
removes all files
.TP
.I de
removes from selected item to the end of the list
.TP
.I ds
remove from selected item to the start of the list
.TP
.I "d1d or dd"
removes selected file only
.TP
.I d4d
removes 4 files starting from selected one
.RE
.TP
.B /
starts a search prompt, write a string and will be showed the next package that contains your string, use ESC to exit
.TP
.B q
quit
.SH NOTES
To use this software you need the portage elog system configured on your
/etc/portage/make.conf, this is a simply configuration:

.nf
# Logging
PORTAGE_ELOG_CLASSES="warn error log"
PORTAGE_ELOG_SYSTEM="save"
.TP
For other details see /usr/share/portage/config/make.conf.example
.SH "REPORTING BUGS"
Report your bugs to our GitHub tracker:
.P
https://github.com/gentoo/elogv/issues
.SH AUTHOR
Luca Marturana (luca89) <lucamarturana@gmail.com>


ChangeLog
# ChangeLog for elogv

0.7.6.6
  * Fix crash for locale without explicit encoding,
      e.g. when running LC_ALL=C elogv (Gentoo bug #527240)
  * Check locale variable LC_ALL during startup, too.
  * Fix resuming from Ctrl-Z/SIGTSTP (Gentoo bug #348110)

0.7.6.5
  * Prevent crash with LC_TIME=en_DK (issue #3)

0.7.6.4
  * Fix crash when log files with bad names are found, only
  * Fix crash for PORT_LOGDIR containing ".log" (issue #4)

0.7.6.3
  * Prevent crash with LC_TIME=ja_JP.UTF-8 (Gentoo bug #464962)

0.7.6.2
  * Make log filename parsing more robust (Gentoo bug #503904)
  * Fix locations of make.conf(.example)
  * Add "make dist" support

0.7.6.1
  * Fix crash for "TERM=vt100 elogv" invokation (Sebastian Pipping <sebastian@pipping.org>

0.7.6
  * Add compression support, thanks to Julian Ospald <hasufell@gentoo.org>

0.7.5 2010-04-26 Luca Marturana <lucamarturana@gmail.com>
  * Added spanish translation

2009-07-13 Luca Marturana <lucamarturana@gmail.com>
	* elogv: fixed bug #262966 on Gentoo Bugzilla, thanks to Davide Pesavento <davidepesa@gmail.com> for reporting and Michael A. Smith <michael@smith-li.com> for the patch
	* setup.py: bump to version 0.7.4

2008-12-07 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: fixed bug #249536 on Gentoo Bugzilla, there was a problem when
	using the dd command, thanks to Arnaud Lech <arnaud@bdefracte.org> for the
	patch
    
2008-11-01 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: fixed bug #236056 on Gentoo Bugzilla, now elogv works fine even
	if the terminal has a black background
	* setup.py: Bump to version 0.7.3

2008-10-18 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: adding patch to display long lines (wrapped), thanks to Douglas
	Anderson <douglasjanderson@gmail.com>

2008-06-01 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Little problem on show_log when the traslation of "Continue..."
	has more than 11 chars.

2008-05-23 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Changed *.clear() with *.erase(), reducing flickering. Also
	added vim modeline, removed trailing spaces from empty lines and other
	minor changes. Thanks to Michał Kiedrowicz <esqualante@o2.pl> for the
	patch.

2008-05-20 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Fixed bug when deleting files from start (ds), thanks to
	Michał Kiedrowicz <esqualante@o2.pl> for the patch

2008-05-16 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Fixed various bugs and added new shortcuts, updated transalations
	and man pages

2008-03-31 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Added localization support ( thanks to Michał Kiedrowicz
	<esqualante@o2.pl> and Christian Faulhammer <opfer@gentoo.org> for the
	German translation), other varius improvements on the code, now the
	list of files can be ordered by class warning level
	* setup.py: Added support to install *.po files and localized manpages

2008-03-21 Luca Marturana (luca89) <lucamarturana@gmail.com> 
	* elogv: Fixed bug #214063 on Gentoo Bugzilla, thanks to Michał Kiedrowicz 
	<esqualante@o2.pl> 

2008-03-05 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Tagging files with the alarm level, thanks to
	Michał Kiedrowicz <esqualante@o2.pl> for the idea and implementation

2008-02-11 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv: Fixed bug on reordering the list of files and fixed wrong j/k keys
	thanks to Michał Kiedrowicz <esqualante@o2.pl>

2008-02-05 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* Makefile: removed
	* setup.py: Now use distutils
	* elogv.py: Fixed bug #195429 on Gentoo Bugzilla and #1800105 on SF.net
	tracker, thanks to Christian Faulhammer <opfer@gentoo.org> and Michał
	Kiedrowicz <esqualante@o2.pl>
	* INSTALL: updated for distutils

2007-09-16 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* Makefile: Release 0.6.1
	* elogv.py: Fixed bug #187595 on Gentoo Bugzilla

2007-06-06 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* Makefile: Release 0.6

2007-05-13 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv.py: don't display version on the in-line help
	* Makefile: applying version of the package to elogv.py isn't required
	anymore
	* INSTALL: added basic installation istructions

2007-05-13 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv.1: Added REPORTING BUGS section
	* Makefile: Added a Makefile
	* makedist.sh: Removed, now we will use the Makefile

2007-05-12 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv.1: Added manpage
	* elogv.py: little performance improvements

2007-05-09 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv.py: new search support

2007-01-29 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* makedist.sh: generate md5 and sign the tarball with gpg

2007-01-02 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* elogv.py: better fix for previous bug

2006-12-14 Luca Marturana (luca89) <lucamarturana@gmail.com>
	* ChangeLog: new format
	* ChangeLog.old: old ChangeLog saved here
	* elogv.py: fixed bug when removing the last entry of the list, thanks Scen
	            for reporting


man/elogv.1.pl
.TH ELOGV "1" "May 2008" "Polecenia użytkownika"
.SH NAZWA
elogv \- przeglądarka plików elog
.SH OPIS
Elogv jest przeglądarką plików elog opartą na curses i pythonie.
.SH UŻYCIE
Możesz sterować programem przy pomocy tych klawiszy:
.TP
.B "Strzałka w dół" lub "j"
przewijanie listy plików w dół o 1 pozycję
.TP
.B "Strzałka w górę" or "k"
przeciwieństwo strzałki w dół
.TP
.B "PageDown"
przewijanie listy w dół o 10 pozycji
.TP
.B "PageUp"
przeciwieństwo PageDown
.TP
.B End
skok do ostatniego pliku na liście
.TP
.B Home
skok do pierwszego pliku na liście
.TP
.B t
uporządkowanie listy plików według dat, najnowszy na górze
.TP
.B a
uporządkowanie listy plików alfabetycznie, najpierw według kategorii, potem (po ponownym wciśnięciu klawisza) według nazwy pakietu
.TP
.B c
uporządkowanie listy plików według typu wiadomości
.TP
.B r
odwrócenie listy plików
.TP
.B SpaceBar
przewijanie wybranego pliku
.TP
.B  h or F1
wyświetlenie pomocy, wciśnij PageUp/PageDown, aby ją przewijać, h lub F1, aby ukryć
.TP
.B d
usuwa wybrane pliki, użycie jest podobne do komendy vima "d", przykłady:
.RS
.TP
.I da
usuwa wszystkie pliki
.TP
.I de
usuwa od wybranego pliku do końca listy
.TP
.I ds
usuwa od wybranego pliku do początku listy
.TP
.I "d1d" lub "dd"
usuwa tylko wybrany plik
.TP
.I d4d
usuwa 4 pliki począwszy od wybranego
.RE
.TP
.B /
włącza tryb wyszukiwania, wpisz ciąg znaków, a zostanie wybrany następny pakiet, który go zawiera, wciśnij ESC, aby zakończyć
.TP
.B q
wyjście
.SH UWAGI
Aby używać tego programu, musisz skonfigurować system elog w /etc/portage/make.conf, oto przykładowa konfiguracja:

.nf
# Logowanie
PORTAGE_ELOG_CLASSES="warn error log"
PORTAGE_ELOG_SYSTEM="save"
.TP
Więcej informacji znajduje się w /usr/share/portage/config/make.conf.example
.SH "ZGŁASZANIE BŁĘDÓW"
Błędy proszę zgłaszać w języku angielskim do naszego serwisu raportowania błędów:

.P
https://github.com/gentoo/elogv/issues
.SH AUTOR
Luca Marturana (luca89) <lucamarturana@gmail.com>
.SH TŁUMACZENIE
Michał Kiedrowicz <esqualante@o2.pl>
